src/test/java/uk/gov/hmcts/reform/sscs/SscsCaseDataUtils.java

```diff
@@ -141,14 +141,16 @@ public static SscsCaseData.SscsCaseDataBuilder builderSscsCaseData(
                 .appeal(appeal)
                 .events(Collections.singletonList(events))
                 .subscriptions(subscriptions)
-                .jointParty("Yes")
-                .jointPartyAddressSameAsAppellant("No")
-                .jointPartyName(JointPartyName.builder().title("mr").firstName("Jon").lastName("Party").build())
-                .jointPartyAddress(Address.builder()
-                        .line1("1 Appellant Ave")
-                        .town("Appellanton")
-                        .county("Appellanty")
-                        .postcode("TS1 1ST")
+                .jointParty(JointParty.builder()
+                        .hasJointParty("Yes")
+                        .jointPartyAddressSameAsAppellant("No")
+                        .name(Name.builder().title("mr").firstName("Jon").lastName("Party").build())
```

Comment:
Minor: Can we newline the properties in the builders to make it easier to read? Applies to all builders.